hoot-core/src/main/cpp/hoot/core/conflate/splitter/DualWaySplitter.h

```diff
@@ -78,6 +80,7 @@ class DualWaySplitter : public OsmMapOperation
   DrivingSide _drivingSide;
   boost::shared_ptr<const OsmMap> _map;
   boost::shared_ptr<OsmMap> _result;
+  std::unordered_set<long> _nodes;
```

Comment:
`unordered_set` has faster inserts that ordered `set` and allows only one copy of each way id.  (Don't want to delete one twice, hoot doesn't like that.)